rest/src/main/java/eu/openg/guessnumberapi/rest/entity/converter/RestResponseConverter.java

```diff
@@ -1,8 +1,5 @@
 package eu.openg.guessnumberapi.rest.entity.converter;
 
-import eu.openg.guessnumberapi.rest.entity.RestGuessResponse;
-import eu.openg.guessnumberapi.usecase.api.BoundaryGuessResponse;
-
-public interface RestResponseConverter {
-    RestGuessResponse convert(BoundaryGuessResponse guessResponse);
+public interface RestResponseConverter<I, O> {
```

Comment:
nice. siūlyčiau šitą į utils modulį, dropinti Rest ir Response iš name ir panaudoti visiems konverteriams